common/src/test/java/org/opensearch/ml/common/output/model/ModelTensorsTest.java

```diff
@@ -28,6 +33,7 @@ public class ModelTensorsTest {
     public ExpectedException exceptionRule = ExpectedException.none();
     private ModelTensors modelTensors;
     private ModelResultFilter modelResultFilter;
+    private Number[] testData;
```

Comment:
Any specific reason to make this variable as class member variable than local variable? It hasn't used anywhere else.